src/Symfony/Component/Validator/Constraints/Email.php

```diff
@@ -96,7 +96,7 @@ public function __construct($options = null)
 
         if ((self::VALIDATION_MODE_STRICT === $this->mode || true === $this->strict) && !class_exists(StrictEmailValidator::class)) {
             // throw new LogicException(sprintf('The "egulias/email-validator" component is required to use the "%s" constraint.', __CLASS__));
```

Comment:
makes sense here as well i guess :)

Reply:
Indeed! I'll do that in a bit :slightly_smiling_face:

Reply:
Done! :slightly_smiling_face: